DeliveryCompany.AppForDrivers/Models/WaybillItem.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace DeliveryCompany.AppForDrivers.Models
+{
+    public enum TypeOfPackages
```

Comment:
I do not really understand this enum. Its like PickupPoint and DeliveryPoint? If yes, than consider naming change, as this one is confising.